versions.yaml

```
@@ -65,5 +68,19 @@ versions:
           gpg: *gpg
           version: '1.14.2'
 
+  - dir: 'exporter'
+    repo: nginx-prometheus-exporter0
```

```suggestion
    repo: nginx1
```

We keep the exporter in the same repo.

---

Fixed.